src/lib/switch/switch.scss

```diff
@@ -12,19 +12,7 @@
   display: none;
 }
 
-:host([disabled]) {
-  @include core.host-disabled;
-
-  .forge-switch {
-    pointer-events: none;
```

Comment:
We may have lost this style during this refactor (related to my comment about disabled on the demo)

Reply:
This was purposeful, to make sure that the disabled cursor is set. I fixed the disabled interactions by only applying active styles to non-disabled or readonly hosts.